internal/tenant/model.go

```diff
@@ -2,35 +2,43 @@ package tenant
 
 import "time"
 
-type TenantInfoList struct {
-	Data []TenantInfoData
+// InfoList defines list of informations about a tenant.
+type InfoList struct {
+	Data []InfoData
 	Meta struct {
 		TotalCount int
 	}
 	Errors []Error `json:"errors"`
 }
-type TenantInfo struct {
-	Data   TenantInfoData
+
+// Info defines a single tenant information.
+type Info struct {
+	Data   InfoData
 	Errors []Error `json:"errors"`
 }
 
+// Error defines an HTTP error.
 type Error struct {
 	Code   string `json:"code"`
 	Detail string `json:"detail"`
 }
 
-type TenantInfoData struct {
+// InfoData is Data used in Info and InfoList.
+type InfoData struct {
 	Attributes Attributes
-	Id         string
+	ID         string
 	Type       string
 }
 
+// Attributes is Attributes used in InfoData.
```

Comment:
See my problem with this is that you are basically not saying anything here. You are explaining something using the same name. I deliberately used 'attributes' as noun in one of my suggestions, since this is different from the type _Attributes_. Personally, I'd go even further and elaborate more, but I guess we can live with this for now.

It is not really valuable to document where a type is used as in "Attributes is Attributes used in InfoData.". What we would be after is that this type entails.